ci/opencpn_rsa.enc

```diff
@@ -0,0 +1,39 @@
+U2FsdGVkX19o9ikDQTBnq0KCiTXKAkKTueToSh+kBes6jZUYljun47tFZXcSCFnh
```

Comment:
Is this file included by purpose?

Reply:
Yes. It's encrypted data used and necessary when uploading built artifacts.